README.md

```diff
@@ -2,30 +2,29 @@
 
 ### Mobile Wallet demo at https://c19.cards
 
-* Click "Connect to Lab and get tested" to retrieve a VC
-* Review details (URLs, DIDs, keys, etc) in Dev Tools
-* Menu > "Scan QR Code" to share your VC with a verifier (e.g., the Verifier demo below)
+- Click "Connect to Lab and get tested" to retrieve a VC
```

Comment:
Is there a reason to replace `*` with `-` here?

Reply:
thanks Josh -- 
It's one of the Markdown linting rules. 

![image](https://user-images.githubusercontent.com/980887/114225546-7122e100-9940-11eb-89bb-5ba37f6d2883.png)

Reply:
Thanks -- the rules are just "don't mix and match" -- they don't prefer `*` to `-` if I'm reading right. But I merged your PR anyway ;-)